### PR TITLE
add rerun visualization

### PR DESCRIPTION
**Pull request by pablovela5620**
This includes the addition of the Segment Anything Model (SAM) as well as ZoeDepth for depth/point cloud generation removing the need for a lidar iphone. Also made it easier to install the enviroment using conda enviroments

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,8 +1,10 @@
-
-Attribution-NonCommercial 4.0 International
+Attribution-NonCommercial 4.0 International (with Rerun additions)
 
 =======================================================================
 
+This work has been modified by Rerun. Additions that belong to Rerun are
+also released under the same license as the original work.
+
```
Comment (nikolausWest) [inline LICENSE]: This file shouldn't be left in it's original form

**Review by nikolausWest — APPROVED** [review]
One thing to fix but ready to merge after

File: demo.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 # This source code is licensed under the license found in the
 # LICENSE file in the root directory of this source tree.
```
Comment (nikolausWest) [inline demo.py:5]: I think this is the place to note that additions have been made that belong to Rerun

File: README.md [hunk 3]
```diff
@@ -23,13 +23,27 @@ This is a PyTorch implementation of [Multiview Compressive Coding (MCC)](https:/
...
+conda env create -f environment.yml
+```
+
+## Quick Rerun visualziation
+<p align="center">
+  <video width="640" height="480" controls>
+    <source src="assets/rerun-example.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+  </video>
+</p>
```
Comment (nikolausWest) [inline README.md]: Actually, this doesn't seem to work. Try doing it like MCC does it. You could upload the video to an issue and link to it instead

**Comment by pablovela5620** [comment]
okay resolved all comments!

**Comment by nikolausWest** [comment]
Great stuff!

**Pull request merged**